Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/utils/UserPreferences.java

```diff
@@ -142,6 +239,129 @@ public static void SavePrefsToIni(Context ctx)
 		// Current CPU core being used. Falls back to interpreter upon error.
 		String currentEmuCore = prefs.getString("cpuCorePref", "0");
 
+		// Current GC modifier range setup. Falls back to 50.000000 upon error.
+		String MainRange_0 = prefs.getString("MainRange_0", "50.000000");
```

Comment:
please use java's naming style conventions, which consist of lower-camelcase (e.g. mainRange0, etc)